realm/realm-library/src/main/java/io/realm/coroutines/RealmFlowFactory.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.coroutines;
+
+import javax.annotation.Nonnull;
+
+import io.realm.DynamicRealm;
+import io.realm.DynamicRealmObject;
+import io.realm.Realm;
+import io.realm.RealmList;
+import io.realm.RealmModel;
+import io.realm.RealmResults;
+import io.realm.annotations.Beta;
+import io.realm.internal.coroutines.InternalFlowFactory;
+import io.realm.rx.CollectionChange;
+import io.realm.rx.ObjectChange;
+import kotlinx.coroutines.flow.Flow;
+
+/**
+ * Factory class used to create coroutine {@link Flow}s.
+ *
+ * This class is used by default unless overridden in {@link io.realm.RealmConfiguration.Builder#flowFactory(FlowFactory}.
+ */
+@Beta
+public class RealmFlowFactory implements FlowFactory {
+
+    private final InternalFlowFactory factory = new InternalFlowFactory(true);
```

Comment:
The argument about frozen should also be exposed as a public constructor for this class, as people cannot otherwise customize it.

Reply:
That's true, I missed this one.